test/configCases/output/inner-dirs-entries/test.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	findBundle: function() {
+		return [
+			"./inner-dir/b.js"
```

Comment:
Also make sure to run `./a.js` so this still works

Reply:
It breaks the test runner because of:
https://github.com/webpack/webpack/blob/master/test/ConfigTestCases.test.js#L208

`currentDirecotry` expected to be the same for all the entries, but it's not true in my test case.
Need to figure out how to fix it.